### PR TITLE
SSCS-10526 in time statement of reasons request

### PR DESCRIPTION
**Pull request by amoslv1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10526

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ z ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/service/SscsCcdConvertService.java [hunk 1]
```diff
@@ -7,6 +7,8 @@
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 import java.util.HashMap;
 import java.util.Map;
+
```
Comment (nickhill111) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/service/SscsCcdConvertService.java:10]: shouldnt be a gap here

Comment (amoslv1) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/service/SscsCcdConvertService.java]: removed

**Review by nickhill111 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/client/CcdClient.java [hunk 2]
```diff
@@ -62,6 +62,7 @@ public StartEventResponse startEvent(IdamTokens idamTokens, Long caseId, String
     }
 
     public CaseDetails submitEventForCaseworker(IdamTokens idamTokens, Long caseId, CaseDataContent caseDataContent) {
+        log.info("Submitting CCD event for caseworker: {}", caseDataContent.getEvent().getDescription());
```
Comment (hajam09) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/client/CcdClient.java:65]: can we add a caseID as well please in the log?

**Review by hajam09 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/client/CcdClient.java (hunk 2 above)
Comment (amoslv1) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/client/CcdClient.java:65]: None of the other logs in the file do that, they don't even specify what the event is...

Comment (amoslv1) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/client/CcdClient.java:65]: Tbh this could just be removed as I had it in for testing

**Review by hajam09 — APPROVED** [review]

**Review by DGordonV1 — APPROVED** [review]

**Pull request merged**